erpnext/support/doctype/issue/issue.py

```diff
@@ -30,16 +34,18 @@ def validate(self):
 
 	def on_update(self):
 		# create the communication email and remove the description
```

Comment:
This comment needs to be edited if we're keeping description.

I think the primary reason why description was removed was to avoid repetition as it would appear in the timeline anyway. Do we have a good reason to keep it now?

Reply:
Answering my own question, I guess you want to keep the description because a task may be created from it subsequently.

How about hiding the field on creating communication? This was we can access it and user needn't read the same thing twice.

Reply:
Descriptions and event timelines give separate pieces of information though.

You could keep updating the issue's description based on its communications, which would be necessary for the task as well.

I think it's fine if we have both visible to the user. What do you think?

Reply:
I guess you're right.. 

@rmehta, @nabinhait Thoughts?